sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -240,16 +240,35 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
 
   /**
    * Loads a JSON file (one object per line) and returns the result as a [[DataFrame]].
+   * With param isStandard we can load multi-lines json object directly.
    * See the documentation on the overloaded `json()` method with varargs for more details.
    *
    * @since 1.4.0
    */
-  def json(path: String): DataFrame = {
+
+  def json(path: String, isStandard: Boolean = false ): DataFrame = {
     // This method ensures that calls that explicit need single argument works, see SPARK-16009
-    json(Seq(path): _*)
+    if (!isStandard) {
+      json(Seq(path): _*)
+    } else {
+      val jsonRDD = sparkSession.sparkContext.wholeTextFiles(path)
+        .map(line => line.toString().replaceAll("\\s+", ""))
+        .map { jsonLine =>
+          val index = jsonLine.indexOf(",")
```

Comment:
maybe this code is bad, I just want to get the json contents
such as: ("filename",json_contents)